packages/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -312,6 +312,7 @@ const Mercury = WebexPlugin.extend({
         this.connected = true;
         this.hasEverConnected = true;
         this._emit('online');
+        this.on('event:featureToggle_update', this.webex.internal.feature.updateFeature);
```

Comment:
Should we have `this.off()` to clear this listener?

Reply:
Maybe in `_onClose` we can clear this listener.
[here](https://github.com/webex/webex-js-sdk/blob/34340d71ce38227676d37ba081d76e57bc3ddff7/packages/%40webex/internal-plugin-mercury/src/mercury.js#L399)

Reply:
Okay. If the event handler will be cleaned up automatically then it's alright.  I just wanted to make sure we dont have unused event handlers.

Reply:
hmm I'm not sure about this.... the event is registered on the `this` instance and `this` is the whole Mercury plugin, so it won't be destroyed when the connection is closed, while this code is executed on every new connection, so I think what will happen is that every time we reconnect mercury connection, we will register the new event handler for 'event:featureToggle_update' and never unregister the old one

Reply:
Thanks, @marcin-bazyl 

It makes sense now. 

@Shreyas281299 - Sorry. I didn't realize I've setup the event listener in connect which is prone to calling more than once.

@marcin-bazyl - I've updated the code to turn on the event handler at the time of `initialize` which is like a Class's constructor in AmpState

Kindly check. Thank you.